source/modules/Git.rst

```diff
@@ -130,7 +130,7 @@ This will make it easier to collaborate. Weirdly enough, collaborating is import
 
 We will do a live demo of setting up a machine for working with this repository now.
```

Comment:
we're going to need to remove these kinds of references to live demos....

I did a video of submitting your work with GitHub, which we _may_ want to reference.

But not one on getting your fork set up -- maybe I should....